src/main/java/com/mojang/datafixers/functions/PointFree.java

```diff
@@ -40,7 +40,12 @@ public final String toString() {
     }
 
     public static String indent(final int level) {
-        return StringUtils.repeat("  ", level);
```

Comment:
`com.google.common.base.Strings#repeat` since we have Guava.

Reply:
Thanks, I changed it